Controllers/getCategories.controller.js

```diff
@@ -0,0 +1,7 @@
+const { selectCategories } = require("../Models/selectCategories.models");
+
+exports.getCategories = (req, res) => {
+  selectCategories().then((categories) => {
+    res.status(200).send({ categories });
+  });
```

Comment:
dont forget to add a catch block to anywhere you are using promises to ensure your error handling is being used :)